rapid7_insightidr/help.md

```diff
@@ -36,6 +36,63 @@ Example input:
 
 ### Actions
 
+#### Query
+
+This action is used to query and return data using LEQL.
```

Comment:
I don't understand this description. It says we're using LEQL, but the input is an ID. Can we be more descriptive around that?

Reply:
We can create a saved query with diferrent endpoint and then use this endpoint to return data with ID of this saved query. 
This description is from JIRA and from API. We can change this description to:
"This action is used to query and return data using LEQL ID"
But this will be different than in API. It will not confuse users?

Reply:
Should we change this?

Reply:
@r7-mgorny I think so - I can see where Joey is coming from. It looks like the action title is more descriptive right now - it gets query results FOR a query ID. Maybe change the description to something like "Get query results for a LEQL query by query ID"?